abowler2/tcl-smart-shopping-list
# Set up router with nav links

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



This work installs `react-router-dom`, sets routes to a view component and an add item component, and puts navigation links at the bottom of the app whose text becomes bold and pink when actively selected.


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |


## Testing Steps / QA Criteria



1. Start the app
2. Note that the view component is visible and that there are links at the bottom for viewing the list and creating an item
3. The View items link should be bold and pink
4. Click on link to create item
5. Note that the view changed to the create item component and the nav link for that view is now bold and pink
